meilisearch/src/analytics/segment_analytics.rs

```diff
@@ -220,9 +220,8 @@ struct Infos {
     ignore_missing_dump: bool,
     ignore_dump_if_db_exists: bool,
     import_snapshot: bool,
-    schedule_snapshot: bool,
+    schedule_snapshot: Option<u64>,
     snapshot_dir: bool,
-    snapshot_interval_sec: u64,
```

Comment:
This PR modifies the `Infos` struct, @gmourier  is that OK?

Reply:
OK; We just have to update this [file](https://github.com/meilisearch/specifications/blob/main/text/0034-telemetry-policies.md) in the associated PR specification.

As it's an `Option<u64>` type, we will receive `none` when the feature is not used, and in the opposite case, the value of the interval (the default one or the value specified by the user), is it right?

Signal to @irevoire

Reply:
> As it's an Option<u64> type, we will receive none when the feature is not used, and in the opposite case, the value of the interval (the default one or the value specified by the user), am I right?

Yes that's correct.

Reply:
This makes me realize that I don't think we made a spec PR for this one. We might have to do so.

Reply:
Sure!

You can track the spec advancements [here](https://github.com/meilisearch/specifications/pull/201) (note: it may be a bit outdated).

I will ask for help regarding the missing specs